pkg/restore/actions/pod_volume_restore_action.go

```diff
@@ -147,15 +147,24 @@ func (a *PodVolumeRestoreAction) Execute(input *velero.RestoreItemActionExecuteI
 	runAsUser, runAsGroup, allowPrivilegeEscalation, secCtx := getSecurityContext(log, config)
 
 	var securityContext corev1.SecurityContext
-	if runAsUser == "" && runAsGroup == "" && allowPrivilegeEscalation == "" && secCtx == "" {
-		securityContext = defaultSecurityCtx()
-	} else {
+	securityContextSet := false
+	// Use securityContext settings from configmap if available
```

Comment:
should we document that admin should be careful about what values they set on this configmap, otherwise it may break non admin users?

Reply:
@kaovilai Well openshift-specific, not nac-specific. Any workload not created by an admin -- or more to the point, any workload that's being assigned a restricted SCC. No NAC code was used to reproduce this bug.